libafl/src/corpus/ondisk.rs

```diff
@@ -161,7 +161,7 @@ where
         &self.current
     }
 
-    /// Current testcase scheduled (mut)
+    /// Current testcase scheduled (mutable)
```

Comment:
I think keeping `mut` for all of these is fine, it should be clear what it means? But open to change it, too

Reply:
I found several occurrences of `(mutable)` before `(mut)` so I began to change them in order to have a consistent syntax.

However I do agree this is not wrong.

Don't mind removing these changes